.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: CI
+on:
+  pull_request:
+  push:
+  schedule:
+    # 2am EST every Saturday
+    - cron: '0 7 * * 6'
+jobs:
+  build_scala2_12:
+    runs-on: ubuntu-latest
+    env:
+      JAVA_OPTS: -Xms2048M -Xmx2048M -Xss6M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v13
+        with:
+          java-version: "adopt@1.8"
+      - name: Coursier cache
+        uses: coursier/cache-action@v6
+      - name: Build and test
+        run: |
+          sbt -v clean +test +scripted
```

This line could be customized to emulate the original `.travis.yml`. Since this repo doesn't seem to have scripted test or cross building it could be simplified to:

```suggestion
          sbt -v clean test
```